client/ayon_core/pipeline/publish/lib.py

```diff
@@ -20,7 +21,7 @@
     Anatomy
 )
 from ayon_core.pipeline.plugin_discover import DiscoverResult
-
+from ayon_core.pipeline.template_data import get_template_data_with_names
```

Comment:
```suggestion

```

Reply:
Ha, same time! Just fixed the unused imports